src/components/donate/Thanks.js

```diff
@@ -0,0 +1,167 @@
+        {t("IBAN")} :{" "}
+      </Grid>
+      <Grid item xs={6}>
+        {formData.paymentMethod === "sepa" &&
+          IBAN.substring(0, 2) +
+            " ............ " +
+            IBAN.substring(IBAN.length - 5, IBAN.length - 2)}
+      </Grid>
+    </>
+  );
+};
+
+const Card = (formData) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Grid item xs={12}></Grid>
+      <Grid item xs={2}></Grid>
+      <Grid item xs={4}>
+        {t("Payment Method")} :{" "}
+      </Grid>
+      <Grid item xs={6}>
+        {t("Card")}
+      </Grid>
+    </>
+  );
+};
+
+const PayPal = (formData) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Grid item xs={12}></Grid>
+      <Grid item xs={2}></Grid>
+      <Grid item xs={4}>
+        {t("Payment Method")} :{" "}
+      </Grid>
+      <Grid item xs={6}>
+        {t("PayPal")}
+      </Grid>
+    </>
+  );
+};
+const Thanks = (props) => {
+  const { t } = useTranslation();
+  const donateConfig = useCampaignConfig().component.donation;
+  const classes = useStyles();
+  const [formData] = useData();
+
+  return (
+    <Container id="proca-donate" className={classes.container}>
+      <Grid container spacing={1}>
+        <Grid item xs={12}>
+          <CardContent className={classes.cardHeader}>
+            <Typography variant="h4" gutterBottom color="textPrimary">
+              {t("Thank you for your donation!")}
+            </Typography>
+
+            <Grid container spacing={1} alignContent="flex-start">
+              <Grid item xs={12}>
+                <Typography variant="h6">
+                  {t("Donation to WeMove Europe")}
+                </Typography>
+              </Grid>
+
+              <Grid item xs={12}></Grid>
+              <Grid item xs={2}></Grid>
+              <Grid item xs={4}>
+                Name :
+              </Grid>
+              <Grid item xs={6}>
+                {formData.firstname} {formData.lastname}
+              </Grid>
+
+              <Grid item xs={12}></Grid>
+              <Grid item xs={2}></Grid>
+              <Grid item xs={4}>
+                Email :
+              </Grid>
+              <Grid item xs={6}>
+                {formData.email}
+              </Grid>
+
+              <Grid item xs={12}></Grid>
+              <Grid item xs={2}></Grid>
+              <Grid item xs={4}>
+                Amount :{" "}
+              </Grid>
+              <Grid item xs={6}>
+                {Number(formData.amount).toLocaleString()}{" "}
+                {donateConfig.currency.symbol}{" "}
+                {formData.frequency !== "oneoff"
+                  ? t("a " + formData.frequency)
+                  : ""}
+              </Grid>
+
+              <Grid item xs={12}></Grid>
+              <Grid item xs={2}></Grid>
+              <Grid item xs={4}>
+                Date :{" "}
+              </Grid>
+              <Grid item xs={6}>
+                {new Date().toLocaleDateString()}
```

Comment:
Hrm ... I was thinking using the browser was the way to go. 
Hrm, so you're thinking that it would be weird to see numbers and dates in a format that doesn't really match the language of the campaign ?

I guess it comes down to how often we think users will be using a browser in a locale they don't really understand.

I'd vote for using the browser locale - but it's not a strong opinion.

Reply:
to clarify: I was suggesting to use the browser, but tell it the locale to use, eg toLocaleDateString('de-DE')

rationale: you and me are probably using the "out of the box" chromium/firefox, that is en-US.

If I see 02/11 on a page of wemove,  I'm assuming it's the 2nd of november, nor the feb 11.

(broader point about displaying number for currency, the thousands and comma separators might be different between the german and french page, because fuckedy fuck)

Reply:
I'll update #613 to include dates too.